crates/router/src/connector/trustpay.rs

```diff
@@ -668,7 +668,8 @@ impl api::IncomingWebhook for Trustpay {
             (trustpay::CreditDebitIndicator::Dbit, trustpay::WebhookStatus::Chargebacked) => {
                 Ok(api_models::webhooks::IncomingWebhookEvent::DisputeLost)
             }
-            _ => Err(errors::ConnectorError::WebhookEventTypeNotFound).into_report()?,
+
+            _ => Ok(api::IncomingWebhookEvent::EventNotSupported),
```

Comment:
Avoid matching on `_`.

Reply:
So for the _ , should I throw an error stating WebhookEventTypeNotFound?

Reply:
You can match on `(trustpay::CreditDebitIndicator::Crdt | trustpay::CreditDebitIndicator::Dbit, trustpay::WebhookStatus::Unknown)` instead.

```rust
(trustpay::CreditDebitIndicator::Crdt | trustpay::CreditDebitIndicator::Dbit, trustpay::WebhookStatus::Unknown) => Ok(api::IncomingWebhookEvent::EventNotSupported),
```

Reply:
Yes , that I would do but in trustpay connector some webhook status have not been matched , like for example
`(trustpay::CreditDebitIndicator::Crdt trustpay::WebhookStatus::Refunded)` has not been,  and if we remove the  match on `_` , it would throw a compile time error

Reply:
Can you match on all the remaining `trustpay::WebhookStatus` variants explicitly?